spec/board_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/board.rb'
+
+RSpec.describe 'testing the board' do
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
```

Comment:
Since you are using the `Board.new` in all of your tests in this file, you can use `let` so as to not repeat yourself.

```ruby
let(:board) { Board.new }
```

Then reuse the above code inside all `expect`.

- [ ] Please use `let` to DRY up your code.